client/state/themes/selectors.js

```diff
@@ -27,6 +27,26 @@ import {
 import { DEFAULT_THEME_QUERY } from './constants';
 
 /**
+ * When wpcom themes are installed on Jetpack sites, the
+ * theme id is suffixed with -wpcom. Some operations require
+ * the use of this suffixed ID. This util function add the
+ * suffix if the theme is a wpcom theme and the site is Jetpack.
+ * or not, given siteId and ThemeId.
+ *
+ * @param {Object} state	Global state tree
+ * @param {Number} siteId	Site ID
+ * @param {String} themeId	Theme ID
+ * @return {String} 		Potentially suffixed theme ID
+ */
+const getSuffixedThemeId = ( state, siteId, themeId ) => {
```

Comment:
Minor nit --  `isThemeActive`, `isInstallingTheme`, and `isPurchasedTheme` have `siteId` and `themeId` args reversed. Should we aim for consistency with them, or any other selectors in this file? 

(Overall consistency would obviously be a good thing -- I think a number of selectors work with a default for `siteId`, so I guess that should normally go last since I can't think of any case where the same would be true for `themeId`.)